aws/cloudformation/application.yaml

```
@@ -0,0 +1,332 @@
+  CloudFrontDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Origins:
+          - Id: S3Origin
+            DomainName: !Sub 
+              - '${Bucket}.s3.${AWS::Region}.amazonaws.com'
+              - Bucket:
+                  Fn::ImportValue: !Sub '${InfrastructureStackName}-FrontendBucket'
+            S3OriginConfig:
+              OriginAccessIdentity: !Sub 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)